cmd/monitor/app/app.go

```diff
@@ -18,9 +18,10 @@ import (
 )
 
 type App struct {
-	RestConfig *rest.Config
-	Logger     *zap.Logger
-	Options    Options
+	RestConfig        *rest.Config
+	Logger            *zap.Logger
+	Options           Options
+	ServiceDescriptor string
```

Comment:
nit: it would be nicer if this is part of Options and unmarshalled and parsed to from flag

Reply:
I second that. And `App` should not be parsing it from string, it should accept a typed object.